README.md

```diff
@@ -190,6 +190,17 @@ models:
           echo "hi"
           ```
 
+### Setting up Azure OpenAI endpoint
```

Comment:
nit: capitalization & name
change to "Setting Up Azure OpenAI"

Reply:
Fixed, thanks for the review.